ral-facilities/datagateway
# Unit tests for settings file fixed #42

## Description
datagateway-table-settings.json was renamed to datagateway-table-settings.example.json
and failing tests in datagateway-table related to the name change were fixed.

## Testing instructions
Add a set up instructions describing how the reviewer should test the code

- [x] Check that tests run successfully and review changes to test coverage
Agile

## Agile board tracking
connect to #42
